lib/pages/detail_movie_page.dart

```diff
@@ -1,33 +1,45 @@
 import 'package:flutter/material.dart';
 
-import 'search_engine.dart';
-import 'styles.dart';
-import 'button_section.dart';
-import 'movie.dart';
-import 'movie_details.dart';
-import 'movie_genres.dart';
-import 'movie_header.dart';
-import 'movie_presentation.dart';
+import '../Widgets/search_engine.dart';
+import '../utils/styles.dart';
+import '../Widgets/button_section.dart';
+import '../models/movie.dart';
+import '../Widgets/movie_details.dart';
+import '../Widgets/movie_header.dart';
+import '../Widgets/movie_presentation.dart';
```

Comment:
Beware of the capital `W` in the imports of the `widgets` folder!

Reply:
Already did the refactor for some earlier comment about the capital in the folder name, but it seems to not changed on every reference 😢 , Done!